apps/google-analytics-4/frontend/src/helpers/signed-requests.ts

```diff
@@ -0,0 +1,38 @@
+import { CreateAppSignedRequestProps } from 'contentful-management/dist/typings/entities/app-signed-request';
+import { PlainClientAPI } from 'contentful-management/dist/typings/plain/common-types';
+
+async function makeSignedRequest(
+  url: URL,
+  appDefinitionId: string,
+  cma: PlainClientAPI,
+  method: CreateAppSignedRequestProps['method'] = 'GET',
+  headers: {} = {} // TOOD: improve HTTP request headers typing
```

Comment:
If we haven't already let's capture this TODO in a subtask (because I think we should fix this and don't want to file it as "someday")

Reply:
Okay I think this is better:

![image](https://user-images.githubusercontent.com/492573/215488646-5555156e-69db-456e-b4a4-12bd58d409e0.png)

![image](https://user-images.githubusercontent.com/492573/215488596-88f63928-c048-4f31-878c-85a73f110ded.png)